src/rsz/src/SteinerTree.cc

```diff
@@ -65,9 +65,27 @@ static void connectedPins(const Net* net,
 SteinerTree* Resizer::makeSteinerTree(const Pin* drvr_pin)
 {
   Network* sdc_network = network_->sdcNetwork();
-  Net* net = network_->isTopLevelPort(drvr_pin)
-                 ? network_->net(network_->term(drvr_pin))
-                 : network_->net(drvr_pin);
+
+  /*
+    Handle hierarchy. Make sure all traversal on dbNets.
+   */
+  odb::dbITerm* iterm;
+  odb::dbBTerm* bterm;
+  odb::dbModITerm* moditerm;
+  odb::dbModBTerm* modbterm;
+
+  db_network_->staToDb(drvr_pin, iterm, bterm, moditerm, modbterm);
+
+  odb::dbNet* db_net;
+  odb::dbModNet* db_mod_net;
+  db_network_->net(drvr_pin, db_net, db_mod_net);
```

Comment:
This pattern repeats a lot.  It seems better to just make an api like `odb::dbNet* db_net = db_network_->flat_net(drvr_pin);`

Reply:
Good idea. I called the api flatNet. Two versions: one for net and one for pin.

Reply:
The same pattern seems to be used with getting pins (am often getting the "flat pin" or "hierarchical pin") so have adde flatPin, hierPin too, in attempt to simplify code